content/participants/ttjc@Harshita-Kanal/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Harshita-Kanal
+type: "participant"
+name: "Harshita Kanal"
+portfolio: "https://github.com/Harshita-Kanal"
```

Comment:
Hey! @Harshita-Kanal you need to add a link to your portfolio website here. If you don't have one right now just leave it blank.